src/_11ty/shortcodes/ArticleCard.js

```diff
@@ -7,7 +7,7 @@ const truncate = require(`${filtersDir}/truncate.js`);
 module.exports = ({ data = {}, templateContent, url } = {}, { label } = {}) => {
   const { author = '', date, image, title = '', locale = '' } = data;
   const imageSrc = image
-    ? image
+    ? `${url}images/${image}`
```

Comment:
@Pete-Robelou I wonder whether here (and in`blog-post`) we just concat `${url}${image}`, which is perhaps a more intuitive outcome for those filling out `image:` i.e. using the path to wherever their image is relative to the blog post itself?

I.e. We're proposing the convention is always to put blog images in an `images` directory (which thankfully the WP export has followed suit), but does it give us more predictability/flexibility, should someone not follow our advice in the README? I.e. And just puts a single image alongside the `index.md`? Should that be fair game?

Reply:
Yeah, that's fair. I'll remove our enforced `/images`.